packages/dialect-react-ui/components/Chat/screens/ThreadPage/Settings.tsx

```diff
@@ -69,6 +72,7 @@ const Settings = ({ onCloseRequest }: SettingsProps) => {
               // TODO: properly wait for the deletion
               onCloseRequest?.();
               setDialectAddress('');
+              navigate(RouteName.Main, { sub: { name: MainRouteName.Thread } });
```

Comment:
We could use navigation shortcuts from `components/Chat/navigation.ts` here like `showMain` to encapsulate the actual navigation considering we might change router in near future. What do you think?

Reply:
Sure, let's start updating to this!